### ISSUE TITLE
Doesn't handle empty `bbox` array.

### ISSUE THREAD

**Issue by mylogon341** [kiliankoe/GeoJSON#5]
Let me start with 'I really like this library. It's helped me a lot'.

Now..
For reasons unknown to me, the api I use returns an empty `bbox` array property. It looks a bit like this
```json
{
    "type": "FeatureCollection",
    "features": [
      {
        "type": "Feature",
        "id": "1",
        "geometry": {
          "type": "Polygon",
          "coordinates": [
            [
              [
                -13.330369591712952,
                29.277490131811966
              ],
              [
                -13.330053091049194,
                29.277490131811966
              ],
              [
                -13.330053091049194,
                29.27774280695788
              ],
              [
                -13.330369591712952,
                29.27774280695788
              ],
              [
                -13.330369591712952,
                29.277490131811966
              ]
            ]
          ]
        },
        "properties": {
          "start": "2022-06-14T14:53:04Z",
          "end": "2022-06-14T15:08:04Z"
        },
        "bbox": []
      }
    ],
    "bbox": []
  }
```

however when I go to decode it, I get the following error
`GeoJSON.BoundingBox.Error.unexpectedValueCount`
which means the whole decoding fails.
Are you able to advise on how to handle this please?

Thanks a lot.
Luke

**Comment by kiliankoe** [comment]
Hey Luke!

That response is interesting, I haven't seen bounding boxes used like that before. I'm not quite sure how to interpret that based on the [relevant section in the spec](https://datatracker.ietf.org/doc/html/rfc7946#section-5), where it says
> The value of the bbox member MUST be an array of length 2*n where n is the number of dimensions represented in the contained geometries, [...]

I mean theoretically it is compliant in that `n` in this case is zero 😅 Right now I'm only supporting 2D or 3D coordinates there. Adding a special case for zero values feels like it might make sense, I'll see about doing that.

Did you mean to close the issue again in any case?

**Comment by mylogon341** [comment]
I closed it because I thought it was something I was doing wrong. I was able to get the api to respond with bbox by constructing a box and posting my feature with it. But it turns out that the api is used by others and the api doesn’t enforce bbox to have values so is frequently empty. 
With this in mind I ended up forking the repo and adding a case to the switch that just gave a 0,0 value if the received value was empty. Not ideal, but I don’t actually use that value so it didn’t hurt my feelings too much to have it assign a “valid” bbox rather than putting more brain power into another option. 

As I side note I think it would also be kind of a nice addition to have a quick calculation to get the bbox from a feature. 

When I’m back at my desk tomorrow, I’ll show examples of what I did, but it’s all pretty basic stuff to get me over the hump. 

Thanks
